<?xml version="1.0"?>
<launch>

 <!-- joy node -->
  <node respawn="false" pkg="joy" type="joy_node" name="joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>


  <node name="robospect_pad" pkg="robospect_pad" type="robospect_pad_node" output="screen">
	<!-- Parameters -->
  	<param name="cmd_topic_vel" value="/robospect_platform_controller/ackermann_cmd" type="string"/>
  	<param name="cmd_service_io" value="modbus_io/write_digital_output" type="string"/>
  	<rosparam command="load" file="$(find robospect_pad)/launch/ps3.yaml" />
  </node>

</launch>
